homeassistant/components/swiss_public_transport/config_flow.py

```diff
@@ -72,14 +90,15 @@ async def async_step_user(
 
     async def async_step_import(self, import_input: dict[str, Any]) -> ConfigFlowResult:
         """Async import step to set up the connection."""
-        await self.async_set_unique_id(
-            f"{import_input[CONF_START]} {import_input[CONF_DESTINATION]}"
-        )
+        unique_id = unique_id_from_config(MappingProxyType(import_input))
```

Comment:
I suggest you make a preliminary PR to remove the import first as it should be removed from next version (so we can skip this completely)

Reply:
true

Reply:
Let me know the PR# when you have it or I'll make one probably tomorrow

Reply:
Ok, you will probably be faster than me :)

Reply:
#119813